lib/linguist/languages.yml

```diff
@@ -1036,6 +1036,18 @@ Frege:
   - .fr
   tm_scope: source.haskell
   ace_mode: haskell
+  
+FreeMarker:
+  type: programming
+  color: "#0050b2"
+  group: Java
```

Comment:
This means that FreeMarker files will be counted as Java files.
Is that what you want?